internal/util/metrics/metrics.go

```diff
@@ -62,13 +83,76 @@ func Init() {
 		[]string{"origin", "origin_type", "method", "status", "http_status", "path"},
 	)
 
+	CacheObjectOperations = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "trickster_cache_operations_objects",
```

Comment:
All `Counter` type metrics should end in `_total`. How about `trickster_cache_object_operations_total`.

I might also define a couple of consts in the package. One for the Namespace, and one for the subsystem.

```go
const (
  metricNamespace = "trickster"
  metricSubsystem = "cache"
)
...

prometheus.CounterOpts{
  Namespace: metricNamespace,
  Subsystem: metricSubsystem,
  Name: "object_operations_total",
...
```